README.md

```diff
@@ -90,21 +90,18 @@ fastify.register(require('fastify-multipart'), {
 });
 ```
 
-If you do set upload limits, be sure to catch the error. An error or exception will occur if a limit is reached. These events are documented in more detail [here](https://github.com/mscdex/busboy#busboy-special-events).
```

Comment:
why was this removed?

Reply:
Because the error is not thrown actually, check this comment: https://github.com/fastify/fastify-multipart/issues/250#issuecomment-888356293